pkg/autoscaler/autoscaler/pdplan_autoscaler.go

```diff
@@ -84,9 +87,34 @@ func (am *autoScalerManager) deleteAutoscalingClusters(tc *v1alpha1.TidbCluster,
 	return nil
 }
 
-func (am *autoScalerManager) updateAutoscalingClusters(tac *v1alpha1.TidbClusterAutoScaler, groups []string, groupTcMap map[string]*v1alpha1.TidbCluster, groupPlanMap map[string]pdapi.Plan) error {
-	// TODO in next PR
-	return nil
+func (am *autoScalerManager) updateAutoscalingClusters(tac *v1alpha1.TidbClusterAutoScaler, groupsToUpdate []string, groupTcMap map[string]*v1alpha1.TidbCluster, groupPlanMap map[string]pdapi.Plan) error {
+	var errs []error
+	for _, group := range groupsToUpdate {
+		actual, oldTc, plan := groupTcMap[group].DeepCopy(), groupTcMap[group], groupPlanMap[group]
+
+		switch plan.Component {
+		case v1alpha1.TiKVMemberType.String():
+			if tac.Spec.TiKV == nil || actual.Spec.TiKV.Replicas == int32(plan.Count) {
+				continue
+			}
+			actual.Spec.TiKV.Replicas = int32(plan.Count)
+		case v1alpha1.TiDBMemberType.String():
+			if tac.Spec.TiDB == nil || actual.Spec.TiDB.Replicas == int32(plan.Count) {
+				continue
+			}
+			actual.Spec.TiDB.Replicas = int32(plan.Count)
+		default:
+			errs = append(errs, fmt.Errorf("unexpected component %s for group %s in autoscaling plan", plan.Component, group))
+			continue
+		}
+
```

Comment:
We should also check and update TidbMonitor here if necessary.

Reply:
We should add a function `syncMonitor` to sync the TidbMonitor out of the TC sync function.